src/filter.ts

```diff
@@ -372,32 +416,31 @@ export class Filter {
    *   }
    * ];
    */
-  column(column) {
-    if (!is.object(column)) {
-      column = {
-        name: column,
+  column(column: RegExp | string | {}): void {
+    let col: Column;
+    if (typeof column !== 'object') {
+      col = {
+        name: column as string,
       };
+    } else {
+      col = column as Column;
```

Comment:
Achtung! It does not work for `RegExp` as you expect.

```
> const is = require('is')
undefined
> r = new RegExp(/.../)
/.../
> is.object(r)
false
> typeof r
'object'
> is.regexp(r)
true
```

Reply:
Super nice catch.  Fixed!